src/codeu/chat/client/commandline/Chat.java

```diff
@@ -181,6 +184,19 @@ private UserContext findUser(String name) {
       }
     });
 
+    // Version Check 
+    // adds a new command "info" and will display the version it is in 
+    panel.register("info", new Panel.Command() {
```

Comment:
Change the "info" command to something like "version" or "server-version" to make it more clear as to what the command will return.